node normalization

cbizon opened the pull request:
Now normalizing output.

Also fixed some of the return types to fit 1.0 better.

PhillipsOwen commented:
node normalization fails with html error 422 when calling graph mode using "famcov_new.json".

requirements.txt is specifying an invalid repo location for reasoner-pydantic (should be using the TranslatorSRI repo).

cbizon commented:
Fixed these issues.

The problem with nn was because the old redis db does not include the biolink: prefixes on predicates.  I added some shimming in there, which we can remove once the databases are up to date.

@PhillipsOwen take a look
